.github/workflows/publish.yml

```diff
@@ -95,6 +95,13 @@ jobs:
 
   publish-docker:
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
@mxyng anything here that might not work?

Reply:
This looks right to me